core/src/scene/sceneLoader.cpp

```diff
@@ -987,6 +988,15 @@ void SceneLoader::loadSource(const std::shared_ptr<Platform>& platform, const st
     if (auto maxZoomNode = source["max_zoom"]) {
         maxZoom = maxZoomNode.as<int32_t>(maxZoom);
     }
+    if (auto tileSizeNode = source["tile_size"]) {
+        const auto BaseTileSize = 256;
+        auto tileSize = tileSizeNode.as<int32_t>();
+        if (tileSize && !( tileSize & (tileSize - 1))) {
```

Comment:
@karimnaaji will use the one you mentioned. @matteblair LOG statement below in the same context, one can easily tell what this statement is checking.